puan/logic/plog/__init__.py

```diff
@@ -566,40 +568,42 @@ def to_dict(self) -> typing.Dict[str, list]:
 
```

Comment:
Change the above from 
The value is a list of three elements:
            1. sign of coeffs (e.g. sign=1 means a+b+c, sign=-1 means -a-b-c)
            2. sub propositions / variables (e.g. a,b,c)
            3. value of support vector (e.g. 3 as in a+b+c>=3)

To

The value is a list of three elements:
 
#. sign of coeffs (e.g. sign=1 means a+b+c, sign=-1 means -a-b-c)
#. sub propositions / variables (e.g. a,b,c)
#. value of support vector (e.g. 3 as in a+b+c>=3)

(Make sure there is a new line before and after the numbered list)